content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -1177,16 +1177,12 @@ Defines a single-field ID on the model.
 
 #### Arguments
 
-| Name  | Required | Type     | Description |
-| ----- | -------- | -------- | ----------- |
-| `map` | **No**   | `String` |             |
-
-**Note:** In version 3.5.0 and later, new `length` and `sort` arguments are available as a Preview feature. See [Index configuration](../../../../concepts/components/prisma-schema/indexes) for more details.
```

Comment:
The link to "Index Configuration" was useful here. Can we get that back maybe somehow?